src/devices/StUsb4500/Objects/ObjectBase.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Iot.Device.StUsb4500.Objects
+{
+    /// <summary>
+    /// Base class for shared object logic.
+    /// </summary>
+    public abstract class ObjectBase
+    {
+        /// <summary>Gets or sets the value.</summary>
+        public uint Value { get; protected set; }
```

Comment:
what's the purpose of this value?

Reply:
@krwq  "objects" transfered to/from the StUsb4500 are based on one "value" which is stored in this base-property and can be bitwise-edited via the methods below.

Reply:
I think the open question is more: do you need this property to be publicly exposed? Or should it be an internal property only?